recipes/django-rosetta/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "django-rosetta" %}
+{% set version = "0.9.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0315a6406818c51f0f0fe04aa30328e802c8befcab1d3a34a242961284cb3387
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Does this need a C compiler?

Reply:
I make the same mistake as in django-parler. Kindly remove please (or I will, if you need me to)

Reply:
Please remove it

Reply:
removed